pkgs/applications/networking/feedreaders/newsboat/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchurl, stfl, sqlite, curl, gettext, pkgconfig, libxml2, json_c, ncurses, makeWrapper
+, asciidoc, docbook_xml_dtd_45, libxslt, docbook_xml_xslt }:
+
+stdenv.mkDerivation rec {
+  name = "newsboat-${version}";
+  version = "2.10.2";
+
+  src = fetchurl {
+    url = "https://newsboat.org/releases/${version}/${name}.tar.xz";
+    sha256 = "1x4nxx1kvmrcm8jy73dvg56h4z15y3sach2vr13cw8rsbi7v99px";
+  };
+
+  prePatch = ''
+    substituteInPlace Makefile --replace "|| true" ""
+  '';
+
+  nativeBuildInputs = [ asciidoc docbook_xml_dtd_45 libxslt docbook_xml_xslt ];
+
+  buildInputs
+    = [ stfl sqlite curl gettext pkgconfig libxml2 json_c ncurses ]
+      ++ stdenv.lib.optional stdenv.isDarwin makeWrapper;
```

Comment:
`makeWrapper` and `pkgconfig` can be treated as native

Reply:
fixed